prow/config/config.go

```diff
@@ -266,6 +269,42 @@ func parseConfig(c *Config) error {
 		}
 	}
 
+	if c.Plank.DefaultDecorationConfig == nil {
+		return errors.New("no default decoration config provided for plank")
```

Comment:
I don't think we should require a default, especially if no jobs are configured for decoration.